frontend/src/packages/auth/auth.ts

```diff
@@ -11,3 +11,4 @@ const authApi = new AuthApi({
 });
 
 export { authApi };
+export { type UserAuthResponseDto } from './libs/types/types.js';
```

Comment:
Should be moved to a special reexport file (src/libs/types/user/user.ts), then reexport it from src/libs/types/user.ts in src/libs/types/types.ts and import it always from there. All these reexports are required foe clean importing routes

Reply:
Oh, also, reexport it in frontend/src/packages/auth/libs/types and use wherever you need. Do not export type from the place it should not be exported (frontend/src/packages/auth/auth.ts)
@andanf-e right?

Reply:
Actually it may be reexported from domain root reexport file, but in our case it should be not `auth` but `users` package - so on `shared` folder it should be declared in `packages/users/libs/types` and in `frontend` - should be reexported also from `users`